Detectors/TRD/base/include/TRDBase/PadParameters.h

```diff
@@ -49,14 +50,14 @@ class PadParameters
   T getValue(int col, int row) const { return getValue(getChannel(col, row)); };
   void setValue(int ich, T value) { mData[ich] = value; }
   void setValue(int col, int row, T value) { setValue(getChannel(col, row), value); }
-  void debug() { std::cout << "in padparameters debug with this:" << this << " with mPlane:mChamber:mNrows:mNchannels::" << mPlane << ":" << mNrows << ":" << mNchannels << "   and a vector size of : " << mData.size() << std::endl; };
+  void debug() { LOG(debug) << "in padparameters debug with this:" << this << " with mPlane:mChamber:mNrows:mNchannels::" << mPlane << ":" << mNrows << ":" << mNchannels << "   and a vector size of : " << mData.size(); };
```

Comment:
do you need this in the header? best to avoid including logging header here.

Reply:
no that was just a temp debugging thing late one night that slipped through, will remove.

Reply:
to remove the logging I need to create a source file and remove the functions at the bottom of the header file that have logging in them as well.

Reply:
removed

Reply:
removed